kbs/quickstart.md

```diff
@@ -179,4 +179,39 @@ Where `/path/to/policy` should be replaced by the real path to your policy file.
 
 Resource policy also needs to be the `rego` syntax defined by [Open Policy Agent](https://www.openpolicyagent.org/).
 
-You can read the notes of [default resource policy file](./src/api/src/policy_engine/opa/default_policy.rego) for more details of resource policy.
\ No newline at end of file
+You can read the notes of [default resource policy file](./src/api/src/policy_engine/opa/default_policy.rego) for more details of resource policy.
+
+## Attestation Token Certificate
+
+You can configure certificate of the signing key of Attestation Token (JWT) by config files.
```

Comment:
Can you mention that this is not required if you are using the Builtin AS (assuming that is true).

Reply:
I'll add some explain about Builtin AS, thanks.